src/main/java/computer_science_basics/homework_1/davit_balabekyan/Task1

```diff
@@ -0,0 +1,11 @@
+Task 1
+* Convert number 456 from decimal to binary = 00000001 11001000
```

Comment:
what is 00000001?

Reply:
0000000111001000 i represented  the value 456 in to 16 bits